examples/vae_text/vae_train.py

```diff
@@ -0,0 +1,477 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Example for building the Variational Autoencoder.
+
+This is an impmentation of Variational Autoencoder for text generation
+
+To run:
+
+$ python vae_train.py
+
+Hyperparameters and data path may be specified in config_trans.py
+
+"""
+
+
+import os
+import sys
+import time
+import argparse
+import importlib
+from typing import Optional, List, Any, Union, Dict, Tuple
+
+import numpy as np
+import torch
+from torch import Tensor
+import torch.nn as nn
+from torch.optim.lr_scheduler import ExponentialLR
+
+import texar.torch as tx
+from texar.torch.modules import BasicRNNDecoderOutput, TransformerDecoderOutput
+from texar.torch.custom import MultivariateNormalDiag
+from texar.torch.modules.decoders.decoder_helpers import Helper
```

Comment:
We don't expose internals to users. I think these imports are all unnecessary, since they can be accessed via `tx.`.

Reply:
Got it.

Reply:
for the `texar.torch.custom`, do we want to include it into modules?